components/UI/titles/categoryTitle.tsx

```diff
@@ -4,16 +4,19 @@ import styles from "../../../styles/components/titles.module.css";
 type CategoryTitleProps = {
   title: string;
   subtitle: string;
+  mobileTitle?: string;
 };
 
 const CategoryTitle: FunctionComponent<CategoryTitleProps> = ({
   title,
   subtitle,
+  mobileTitle = title,
```

Comment:
I don't understand why we should have different titles here. 

It was in the design ?

Reply:
As the issues and figma were about design for mobile, I tried to impact the desktop version as little as possible (except when it was obvious that I had to apply the design to the desktop version too). For this example, the current text on the site was not the same as the one on the figma. As the figma only covered the mobile part, I only modified the title on mobile. But perhaps I misjudged the extent to which I shouldn't affect the desktop version. So should I keep like that, or update using the figma's text also on desktop (and thus remove the "mobileTitle" property in the component)

Reply:
Imo you should just not follow figma on that and let one title for both (take the desktop one please)